ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -48,7 +49,8 @@ type ssh_daemon_options_type = {
     "ClientAliveInterval"               ? long
     "DenyGroups"                        ? string
     "DenyUsers"                         ? string
-    "HostKey"                           ? string
+    "GSSAPIStrictAcceptorCheck"         ? ssh_yesnostring
+    "HostKey"                           ? string[]
```

Comment:
we use the following for per user ssh (client) config:
https://github.com/quattor/configuration-modules-core/blob/master/ncm-metaconfig/src/main/metaconfig/ssh/client_attrs.tt

the boolean handling in the TT can be cleaned up (there's a https://github.com/quattor/configuration-modules-core/blob/master/ncm-metaconfig/src/main/metaconfig/ssh/pan/schema.pan which uses boolean instead of `yesno_string`), but the remaining is indeed hardcoded.

i have nothing against the hardcoding in the perl code, it's unavoidable due to crappy config formats. i was just wondering if the backwards incompatibility in the schema was really needed).
i'm in favour of handling multi valued options with `string[]` rather than some pan string joining.